### PR TITLE
Fix part #575: Low fi Home Page display Profile name

### PR DESCRIPTION
**Pull request by veena14cs**
## Explanation
This PR displays the name of the profile on home page.

### Mock
https://xd.adobe.com/view/0687f00c-695b-437a-79a6-688e7f4f7552-70b6/screen/cb72aed9-af32-4ab6-9a80-394d9b30f6a3/HP-Home-Page-1

### Screenshot
Screenshot_20200130-115826

## Checklist

- [ ] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

File: app/src/sharedTest/java/org/oppia/app/home/HomeActivityTest.kt [hunk 1]
```diff
@@ -85,6 +97,27 @@ class HomeActivityTest {
     Intents.release()
   }
 
+
```
Comment (rt4914) [inline app/src/sharedTest/java/org/oppia/app/home/HomeActivityTest.kt]: Remove this extra line.

File: app/src/sharedTest/java/org/oppia/app/home/HomeActivityTest.kt [hunk 2]
```diff
@@ -320,21 +353,24 @@ class HomeActivityTest {
...
+    launch<HomeActivity>(createHomeActivityIntent(0)).use {
       pressBack()
       onView(withText(R.string.home_activity_back_dialog_exit)).perform(click())
       intended(hasComponent(ProfileActivity::class.java.name))
     }
   }
 
+  private fun createHomeActivityIntent(profileId: Int): Intent {
+    return HomeActivity.createHomeActivity(ApplicationProvider.getApplicationContext(), profileId)
+  }
```
Comment (rt4914) [inline app/src/sharedTest/java/org/oppia/app/home/HomeActivityTest.kt:372]: Add one blank line after this

**Review by rt4914 — APPROVED** [review]
LGTM, thanks

File: app/src/sharedTest/java/org/oppia/app/home/HomeActivityTest.kt (hunk 1 above)
Comment (nikitamarysolomanpvt) [reply app/src/sharedTest/java/org/oppia/app/home/HomeActivityTest.kt]: +1

**Review by nikitamarysolomanpvt — APPROVED** [review]
LGTM

File: app/src/sharedTest/java/org/oppia/app/home/HomeActivityTest.kt (hunk 1 above)
Comment (veena14cs) [reply app/src/sharedTest/java/org/oppia/app/home/HomeActivityTest.kt]: done.

File: app/src/sharedTest/java/org/oppia/app/home/HomeActivityTest.kt (hunk 2 above)
Comment (veena14cs) [reply app/src/sharedTest/java/org/oppia/app/home/HomeActivityTest.kt:372]: Done.

**Comment by nikitamarysolomanpvt** [comment]
Please create a PR for fixing spacing of name displayed... getting cut-out when bigger.

**Comment by veena14cs** [comment]
> Please create a PR for fixing spacing of name displayed... getting cut-out when bigger.

There is issue related to this #582.

**Pull request merged**